terraform/alb.tf

```diff
@@ -0,0 +1,76 @@
+resource "aws_alb" "main" {
```

Comment:
@zackads @Ninamma @stuart-madetech See terraform provider: https://registry.terraform.io/providers/hashicorp/aws/latest/docs

This is the go-to resource for looking up available AWS terraform resources and the inputs required and outputs from each resource.

Also of use: https://www.terraform.io/docs/language/resources/index.html